.devcontainer/setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+echo "--- CONFIURING CODESPACE ---"
+
+# general setup
+sudo apt-get install htop vim -y
+source $NVM_DIR/nvm.sh
+nvm install v16
+nvm alias default 16
+
+# configure redis
+sudo apt-get install redis-tools -y
+docker run -p 6379:6379 --name redis -d redis
+
+# configure postgres
+sudo apt-get install postgresql-client -y
+docker volume create pg-data
+docker run -p 5432:5432 --name postgres -v pg-data:/var/lib/postgresql/data -e POSTGRES_HOST_AUTH_METHOD=trust -e POSTGRES_USER=codespace -e POSTGRES_PASSWORD="" -e POSTGRES_DB=grouparoo_development -d postgres
+
+# configure node
+npm install -g pnpm
+pnpm install --unsafe-perm
+
+# setup apps
+touch ~/.zshrc
+echo "alias roo=/workspaces/grouparoo/cli/dist/grouparoo.js" >> ~/.zshrc
```

Comment:
By default it seems my codespace terminal is using Bash instead of zsh, so I had to add it to `~/.bashrc` instead.

Reply:
Dang! I guess it's person-specific :.

Reply:
Maybe add it to both?